src/index.js

```diff
@@ -203,14 +225,30 @@ export default class ImageTool {
   }
 
   /**
-   * Makes buttons with tunes: add background, add border, stretch image
+   * Returns configuration for block tunes: add background, add border, stretch image
    *
    * @public
    *
-   * @returns {Element}
+   * @returns {Array}
    */
   renderSettings() {
-    return this.tunes.render(this.data);
+    const tunes = ImageTool.tunes.concat(this.config.actions);
+
+    return tunes.map(tune => ({
+      icon: tune.icon,
+      label: this.api.i18n.t(tune.title),
+      name: tune.name,
+      toggle: tune.toggle,
+      isActive: this.data[tune.name],
+      onActivate: () => {
+        if (typeof tune.action === 'function') {
```

Comment:
please add a comment describing that this is support for the legacy property ( maybe with a link to https://github.com/editor-js/image/pull/49)